media/js/wiki.js

```diff
@@ -847,5 +847,90 @@
     if($('details').length){
         initDetailsTags();
     }
+    
+    function openJSFiddle(title, htmlCode, cssCode, jsCode) {
+       var $form = $('<form method="post" target="_blank" action="https://jsfiddle.net/api/mdn/">'
```

Comment:
Instead of a large string, let's create the elements with JS.  We retrieve most of them to set their values anyway.